app/assets/javascripts/discourse/app/mixins/card-contents-base.js

```diff
@@ -189,57 +188,30 @@ export default Mixin.create({
     return this._show($target.text().replace(/^@/, ""), $target);
   },
 
-  _positionCard(target, event) {
-    this._popperReference?.destroy();
-
+  _positionCard(target) {
     schedule("afterRender", () => {
       if (!target) {
         return;
       }
 
+      const avatarOverflowSize = 44;
```

Comment:
I know it's not from you, but can we make this a constant?

Reply:
yes